tests/settings.py

```diff
@@ -50,11 +51,17 @@
 USE_L10N = False
 STATIC_URL = '/static/'
 CORS_ORIGIN_ALLOW_ALL = True
+MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
+EMAIL_PORT = '1025'
+MEDIA_URL = '/media/'
+
+# during development only
+EMAIL_BACKEND = 'django.core.mail.backends.console.EmailBackend'
 
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [os.path.join(BASE_DIR, '../django_freeradius/templates')],
```

Comment:
wait a moment.. is this needed? If yes we'll need to update the install instructions too.
If you can get rid of it, it would be better. I guess you need this for the HTML template right?
If that's the case, you can calculate the absolute path of the template and use that as the default value.

Reply:
Yeah. Needing this for the pdf template.